packages/scene-composer/src/components/three-fiber/Fog.spec.tsx

```diff
@@ -0,0 +1,78 @@
+import { render } from '@testing-library/react';
+import * as THREE from 'three';
+jest.doMock('@react-three/fiber', () => {
```

Comment:
Same feedback here for use of `requireActual`. Here's an [example](https://github.com/awslabs/iot-app-kit/blob/228037da8351499be142b75fc6b62aefc9047ee2/packages/scene-composer/src/components/three-fiber/CameraPreview/__tests__/CameraPreview.spec.tsx#L9-L11).

Reply:
Able to get this improvement in.